src/lib.rs

```diff
@@ -442,11 +445,31 @@ pub trait Rng {
     /// use rand::{thread_rng, Rng};
     ///
     /// let mut v = [0u8; 13579];
-    /// thread_rng().fill_bytes(&mut v);
+    /// thread_rng().fill(&mut v);
     /// println!("{:?}", &v[..]);
     /// ```
-    fn fill_bytes(&mut self, dest: &mut [u8]) {
-        impls::fill_bytes_via_u64(self, dest)
+    fn fill(&mut self, dest: &mut [u8]) {
+        impls::fill_via_u64(self, dest)
+    }
+
+    /// Fill `dest` entirely with random data.
+    ///
+    /// Implementations of this trait must implement at least one of
+    /// `next_u32`, `next_u64`, `fill` and `try_fill` directly. In the case this
```

Comment:
I don't think this is true, only `next_u32` has to be implemented? But this part of the documentation is not going to live long anyway.